### PR TITLE
German Language adaptations

### PR DESCRIPTION
**Pull request by enno123**
Missing translations replaced and existing ones adapted
+ OAuth

### CODE REVIEW

**Comment by Derkades** [comment]
Looks good, thanks

File: custom/languages/German/general.php [hunk 1]
```diff
@@ -53,9 +53,9 @@
     'no_messages' => 'Keine Nachrichten',
     'x_new_messages' => 'Du hast {x} neue Nachrichten', // Don't replace {x}
     'loading' => 'Lädt...',
-    'report' => 'Melden',
-    'reported_by' => 'Reported by {x}',
-    'view_report' => 'View report',
+    'report' => 'Report',
```
Comment (TeemoCell) [inline custom/languages/German/general.php:56]: `Melden` was right xD

File: custom/languages/German/general.php [hunk 2]
```diff
@@ -53,9 +53,9 @@
...
+    'reported_by' => 'Reportet von {x}',
```
Comment (TeemoCell) [inline custom/languages/German/general.php:57]: `Gemeldet von {x}`

File: custom/languages/German/general.php [hunk 3]
```diff
@@ -53,9 +53,9 @@
...
+    'view_report' => 'Report ansehen',
```
Comment (TeemoCell) [inline custom/languages/German/general.php:58]: `Meldung ansehen`

File: custom/languages/German/general.php (hunk 2 above)
Comment (Derkades) [reply custom/languages/German/general.php:57]: want to open a PR with these changes?

**Pull request merged**